README.md

```diff
@@ -198,9 +198,11 @@ The helperPod is allowed to run on nodes experiencing disk pressure conditions,
 `sharedFileSystemPath` allows the provisioner to use a filesystem that is mounted on all nodes at the same time.
 In this case all access modes are supported: `ReadWriteOnce`, `ReadOnlyMany` and `ReadWriteMany` for storage claims.
 
+`storageClassConfigs` is a map from storage class names to objects containing `nodePathMap` or `sharedFilesystemPath`, as described above.
+
 In addition `volumeBindingMode: Immediate` can be used in  StorageClass definition.
 
-Please note that `nodePathMap` and `sharedFileSystemPath` are mutually exclusive. If `sharedFileSystemPath` is used, then `nodePathMap` must be set to `[]`.
+Please note that `nodePathMap`, `sharedFileSystemPath`, and `storageClassConfigs` are mutually exclusive. If `sharedFileSystemPath` or `stroageClassConfigs` are used, then `nodePathMap` must be set to `[]`.
```

Comment:
IIUC, the backward compatibility of an existing configmap with `nodePathMap` is also addressed, right?

Reply:
Yes. The ConfigMap under this patch can contain one and only one of nodePathMap, sharedFileSystemPath, or storageClassConfigs. An existing ConfigMap with one of nodePathMap and sharedFileSystemPath should continue to operate as-is, which I believe the tests demonstrate.